File: examples/lua/mega_flow/executor.lua
local M = {}

function M.execute(cmd)
    -- SINK — os.execute · lua.cmdi.os_execute · CWE-78
    os.execute(cmd)
    return cmd
end

function M.clean_twin()
    -- NEGATIVE — same sink kind with a constant argument must not report.
    os.execute("echo clean")
    return "clean"
end

return M

File: examples/swift/mega_flow/Executor.swift
import Foundation

enum Executor {
    // SINK — Process().launch · swift.cmdi.process_launch · CWE-78
    static func execute(_ cmd: String) -> String {
        let proc = Process()
        proc.launchPath = "/bin/sh"
        proc.arguments = ["-c", cmd]
        proc.launch()
        return cmd
    }

    static func cleanTwin() -> String {
        // NEGATIVE — same sink kind with a constant argument must not report.
        let proc = Process()
        proc.launchPath = "/bin/sh"
        proc.arguments = ["-c", "echo clean"]
        proc.launch()
        return "clean"
    }
}
